hr_timesheet_role/models/project_project.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    is_timesheet_role_required = fields.Boolean(
+        string='Timesheet Role Required',
+        default=lambda self: self._default_is_timesheet_role_required(),
+    )
+    limit_timesheet_role_to_assignments = fields.Boolean(
+        string='Limit Timesheet Role to Assignments',
+        default=(
```

Comment:
Why do you make it a tuple?

Reply:
Hm, actually I'm not sure if it's a tuple - there's no `,` in the end thus it should be only expression in parentheses

Reply:
To avoid pylint errors?

Reply:
Yep, for styling